drivers/espi/Kconfig.xec_v2

```diff
@@ -145,4 +145,18 @@ config ESPI_SAF_INIT_PRIORITY
 	help
 	  Driver initialization priority for eSPI SAF driver.
 
+config ESPI_PERIPHERAL_KBC_IBF_EVT_DATA
+	bool "KBC event data format in IBF"
+	default n
```

Comment:
This is the default for booleans, don't write `default n`

Reply:
Fixed